unfetter-discover/unfetter-store
# Issue 855/store ctf

supports unfetter-discover/unfetter#855

fixes a compile warning
gives better names
clean up

## To test
* pull issue-855/store-ctf branches in unfetter and unfetter-store
* `cd unfetter-store/unfetter-ctf-ingest && npm run build` verify no errors or warnings about missing File or PapaParse classes
* restart stack
* make sure app still works
* visit the create workproduct page, import reports from csv, use `unfetter-store/unfetter-ctf-ingest/test-data/ctf-sample.csv`
* `cd unfetter-store/unfetter-ctf-ingest && generate:reports:fromcsv -- -f test-data/reports/ctf-sample.csv`
 * console 1  - `cd unfetter-store/unfetter-ctf-ingest && npm run start`
 * console 2 - `cd unfetter-store/unfetter-ctf-ingest/test-data/reports/curl-sample.sh`, verify no errors
